### PR TITLE
Plain JavaScript with objects

### PR DESCRIPTION
**Pull request by Ali-0111**
**Note:** I don't have coding partner this week.
**Objectives:** 
-  Different ways to create objects in JavaScript.
- Create a collection that keeps a list of books
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection
- Display all books saved in the collection in the top part of the page.
- Make sure that when a user clicks the "Add" button  a new book is added to the collection.
- The new book is displayed in the page.
- Make sure that when a user clicks the "Remove" button:
- The correct book is removed from the collection.
- The correct book dissapears from the page.
- Make sure that data is preserved in the browser's memory by using [localStorage]

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  removeBtn(e) {
+    const li = e.target.parentElement;
+    this.collection = this.collection.filter((book) => book.title !== li.id);
+    this.saveLocal(this.collection);
+    li.remove();
+  }
+
+  createList(element) {
+    // const newBook = this.getfromLocal();
```
Comment (GedeonTS) [inline app.js]: - [ ] Good job creating the book successfully :clap: , However, kindly remove this commented-out line of code to make your code cleaner :recycle:

File: app.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    this.ul.appendChild(li);
+    this.clearPlaceHolder();
+  }
+
+  create() {
+    // DOM  for inputs
+    const newBook = { title: this.title.value, author: this.author.value };
+    this.collection.push(newBook);
+    this.createList(newBook);
+    // return collection;
```
Comment (GedeonTS) [inline app.js]: I think it would be better to remove this commented-out line of code to make your code cleaner :recycle:

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @Ali-0111 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- The Book creation functionality successfully :+1: 
-  The Book deletion functionality successfully :+1: 
- No linter errors :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by osamaashraf6 — APPROVED** [review]
Hi @Ali-0111 👋, 
# STATUS: APPROVED! 🥇🟢🟢💯 


Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it: 
Congratulations! 🎉 

## Highlights💯🎈🪄✔️


- [x] The Book creation functionality is successful ✔️
- [x] The Book deletion functionality is successful ✔️
- [x] No linter errors. ✔️
- [x] Good usage in correcting GitHub flow.✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**